frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/create-local-volume-set.tsx

```diff
@@ -100,6 +101,19 @@ const ConfirmationModal = ({ state, dispatch, setInProgress, setErrorMessage, ns
     dispatch({ type: 'setShowConfirmModal', value: false });
   };
 
+  const description = (
+    <>
+      <span>
+        After the volume set and storage class are created you won&apos;t be able to go back to this
```

Comment:
Please avoid &apos, it creates issues for i18n parser
```suggestion
        After the volume set and storage class are created you won't be able to go back to this
```

Reply:
This gives linter error.

Reply:
You can try wrapping in string, the way it was previously.
```suggestion
        "After the volume set and storage class are created you won't be able to go back to this"
```